src/components/hal-body.tsx

```diff
@@ -1,26 +1,15 @@
 import * as React from 'react';
 import { PageProps } from '../types';
-import { highlightJson } from '../util';
```

Comment:
I guess this function and package dependency can be removed now?

Reply:
Good point on removing the function, but unfortunately the package is still used for syntax highlighting code blocks in markdown bodies!

Reply:
Fixed